### PR TITLE
Latest gradle-baseline

### PR DESCRIPTION
**Pull request by carterkozak**
==COMMIT_MSG==
Latest gradle-baseline
==COMMIT_MSG==

### CODE REVIEW

File: atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraRequestExceptionHandler.java [hunk 1]
```diff
@@ -301,7 +301,8 @@ public boolean shouldBackoff(Exception ex) {
         @Override
         public long getBackoffPeriod(int numberOfAttempts) {
             double randomCoeff = ThreadLocalRandom.current().nextDouble(1 - UNCERTAINTY, 1 + UNCERTAINTY);
-            return (long) (Math.min(BACKOFF_COEFF * Math.pow(2, numberOfAttempts), MAX_EXPECTED_BACKOFF) * randomCoeff);
+            return (long) (Math.min(BACKOFF_COEFF * Math.pow(2, numberOfAttempts), (double) MAX_EXPECTED_BACKOFF)
+                    * randomCoeff);
```
Comment (carterkozak) [inline atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraRequestExceptionHandler.java:305]: From `./gradlew classes testClasses -PerrorProneApply=LongDoubleConversion`

File: atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/thrift/ThriftQueryWeighers.java [hunk 2]
```diff
@@ -80,7 +80,7 @@ public QueryWeight weighSuccess(T result, long timeTakenNanos) {
             @Override
             public QueryWeight weighFailure(Exception error, long timeTakenNanos) {
                 return ImmutableQueryWeight.builder()
-                        .numBytes(ESTIMATED_NUM_BYTES_PER_ROW * numberOfQueriedRows)
+                        .numBytes(ESTIMATED_NUM_BYTES_PER_ROW * ((long) numberOfQueriedRows))
```
Comment (carterkozak) [inline atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/thrift/ThriftQueryWeighers.java:83]: From `./gradlew classes testClasses -PerrorProneApply=NarrowCalculation`

File: atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/Schema.java [hunk 3]
```diff
@@ -323,6 +323,7 @@ public Namespace getNamespace() {
      *
      * @param srcDir root source directory where code generation is performed.
      */
+    @SuppressWarnings("DangerousIdentityKey")
```
Comment (carterkozak) [inline atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/Schema.java:326]: `DangerousIdentityKey` suppressions lifted directly from #5753

File: atlasdb-client/src/main/java/com/palantir/atlasdb/util/SlidingWindowMetricsInvocationHandler.java [hunk 4]
```diff
@@ -72,10 +71,7 @@ public void onSuccess(@Nullable InvocationContext context, @Nullable Object resu
     public void onFailure(@Nullable InvocationContext context, @Nonnull Throwable cause) {
         markGlobalFailure();
         if (context == null) {
-            log.debug(
-                    "Encountered null metric context likely due to exception in preInvocation: {}",
-                    UnsafeArg.of("cause", cause),
-                    cause);
+            log.debug("Encountered null metric context likely due to exception in preInvocation", cause);
```
Comment (carterkozak) [inline atlasdb-client/src/main/java/com/palantir/atlasdb/util/SlidingWindowMetricsInvocationHandler.java:74]: DRY: Logging the cause as a throwable provides everything that cause.toString does, and more!

File: atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/SweepStatsKeyValueService.java [hunk 5]
```diff
@@ -263,7 +263,7 @@ private void flushTask() {
             }
         } catch (Throwable t) {
             if (!Thread.interrupted()) {
-                log.warn("Error occurred while flushing sweep stats: {}", UnsafeArg.of("throwable", t), t);
```
Comment (jeremyk-91) [inline atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/SweepStatsKeyValueService.java:266]: ugh. lol

**Review by jeremyk-91 — APPROVED** [review]
👍

**Pull request merged**